client/libs/material/src/lib/sermon-table/sermon-table.component.scss

```diff
@@ -0,0 +1,47 @@
+@import 'libs/theme/variables.scss';
+@import 'libs/theme/main.scss';
+
+.full-width-table {
+  width: 100%;
+}
+
+.bg-color-primary {
+  background-color: $primary;
+}
+
+.create-btn {
+  min-width: 40px !important;
+}
+
+.create-icon {
+  color: $accent;
+  width: 25px;
+  height: 25px;
+  font-size: 25px;
+}
+
+.actions-container {
+  max-width: 50px !important;
+}
+
+.edit-btn {
+  min-width: 25px !important;
```

Comment:
looks like we are using `min-width: 25px !important;` a lot here.

maybe we can just create one style and apply that where needed?